src/api/schemas/user-schema.ts

```diff
@@ -33,6 +33,8 @@ const countryCode = string({
     invalid_type_error: invalidMessage('Country Code', 'string')
 }).regex(new RegExp('^([0-9]|[1-9][0-9]|[1-9][0-9][0-9])$'), 'Invalid country code (0-999)');
 
+const bio = string({});
```

Comment:
Add a tiny bit more details for `required_error` and `invalid_type_error`. All attributes follow the same structures!

Reply:
Done and done